testgrid/config/config.yaml

```diff
@@ -1933,6 +1933,92 @@ dashboards:
   - name: kubelet-non-cri-1.6
     test_group_name: ci-kubernetes-node-kubelet-non-cri-1.6
 
+# Blocking tests for the next release
+- name: release-master-blocking
+  dashboard_tab:
+  # Build/Verify
+  - name: build-master
+    test_group_name: ci-kubernetes-build
+  - name: verify-master
+    test_group_name: ci-kubernetes-verify-release
+  - name: test-go-master
+    test_group_name: ci-kubernetes-test-go-release
+  - name: federation-build-master
+    test_group_name: ci-kubernetes-federation-build
+  # E2E
+  - name: gce-master
+    test_group_name: ci-kubernetes-e2e-gce-release
```

Comment:
Yeah, thank you for pointing that out. Many of these are broken, but I'm waiting for more consensus before spending time fixing them :)